### ISSUE TITLE
Unable to run Android Emulator error 4294967201

### ISSUE THREAD

**Issue by kyzer-android** [google/android-emulator-hypervisor-driver-for-amd-processors#10]
hello i don't know why silent instalation dosent work for me:

"Android SDK is up to date.
Running Android Emulator Hypervisor Driver for AMD Processors installer
[SC] ControlService echec(s) 1062 :

service don't start.

[SC] DeleteService succeed
fail of [SC] StartService error 4294967201."

Someone know what is error 4294967201?

**Comment by majorvin** [comment]
Any news on this?

**Comment by kyzer-android** [comment]
Hi, I have solved my problem.
For thoses who have the sames issues  you have to make 2 things:
-Activate Virtualization in bios
-Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
& run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
It should be work after a reboot.

**Comment by kyzer-android** [comment]
Hi, I have solved my problem.
For thoses who have the sames issues  you have to make 2 things:
-Activate Virtualization in bios
-Deactivate windows virtualization

**Comment by Afvillegas** [comment]
> 
> 
> Hi, I have solved my problem.
> For thoses who have the sames issues you have to make 2 things:
> -Activate Virtualization in bios
> -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> It should be work after a reboot.

Great. It works!

**Comment by Furiouspenguin** [comment]
> Hi, I have solved my problem.
> For thoses who have the sames issues you have to make 2 things:
> -Activate Virtualization in bios
> -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> It should be work after a reboot.

2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.

**Comment by AlexeyIgnatev** [comment]
> > Hi, I have solved my problem.
> > For thoses who have the sames issues you have to make 2 things:
> > -Activate Virtualization in bios
> > -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> > & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> > It should be work after a reboot.
> 
> 2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.

Hi. for me the problem was in avast antivirus. In settings click Troubleshoot on the left side of the screen, uncheck the box next to Enable hardware assisted virtualization, then click OK to confirm and restart your computer. it worked for me. Excuse me for my English, I'm from Russia myself 👍 :-)

**Comment by Taogle2018** [comment]
@Furiouspenguin What is your antivirus program?

**Comment by Furiouspenguin** [comment]
> > > Hi, I have solved my problem.
> > > For thoses who have the sames issues you have to make 2 things:
> > > -Activate Virtualization in bios
> > > -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> > > & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> > > It should be work after a reboot.
> > 
> > 
> > 2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.
> 
> Hi. for me the problem was in avast antivirus. In settings click Troubleshoot on the left side of the screen, uncheck the box next to Enable hardware assisted virtualization, then click OK to confirm and restart your computer. it worked for me. Excuse me for my English, I'm from Russia myself 👍 :-)

Hi, thanks for the reply! I tried it, but unfortunately, it didn't help in my case... :( I suspect it had something to do with the windows update, since just a few days before I used it fine and I don't think anything else changed with in my system...

**Comment by Taogle2018** [comment]
What is the Windows 10 version? I can have a try to see if I can help

**Comment by josecv21** [comment]
me pasa exactamente lo mismo, ayer se me actualizo windows y hoy ya no me funciona

**Comment by josecv21** [comment]
la version de windows es la 1903 y mirando en el registro de actualizaciones he encotrado esto"todas las actualizaciones de Windows deshabilitarán la característica vGPU de RemoteFX debido a un problema de seguridad."

**Comment by Taogle2018** [comment]
I am wondering whether update of Windows can turn on hyper-V? I heard certain Windows version (perhaps Home edition) would turn on Hyper-V forcefully.
What is the output of "emulator-check.exe hyper-v"?

**Comment by AlexeyIgnatev** [comment]
After windows 10 update three days ago my emulator stopped starting too. If hyper-V and hypervision platform are disabled and svm is enabled then another program uses virtualization

**Comment by Furiouspenguin** [comment]
Thanks for all the replies! (I'm sorry I was away this weekend so I couldn't check my PC)
So my build is (in short): Windows 10 pro (10.0.18363 build 18363) on an AMD Ryzen 9 3900x with x570 motherboard (Asus).

@Taogle2018 my antivirus is Avast!, so I already disabled hardware-assisted virtualization as @AlexeyIgnatev suggested, but sadly it didn't help at all.

Now what I found in the windows update history is a Version 1909 update (although through the link it says 1903... a bit confusing tbh), which led me to:
https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351

Now I'm guessing this RemoteFX vGPU had something to do with my problem, since they clearly state here that any VM that uses it (RemoteFX is enabled in it) will fail when started. So now I guess we have the reason why the update killed off the emulators, but I have no idea how to fix it... Any ideas?

**Comment by Taogle2018** [comment]
@Furiouspenguin I do not know RemoteFX vGPU so I cannot comment anything. But gvm itself does not use/depend on RemoteFX.
The error code showed by "silent_install.bat" or "sc start gvm" means the following exactly: the virtualization extension of the CPU is missing (not your case, I think) or disabled (I will assume a Windows update won't be able to change your BIOS settings), or used by 3rd party software in a way not sharable with gvm. Hyper-V and Avast are two examples but there could be others as well.
I do not have any clue of possible changes made by Windows Update itself.

**Comment by josecv21** [comment]
Solved.... I update Windows 10 to 2004 versión.

**Comment by Furiouspenguin** [comment]
@Taogle2018 Yeah that's why I didn't search there first, but it was the only thing that has changed (and had anything to do with virtual machines). Also the update states that even if it's just enabled in it, the virtual machine won't work, so that was probably it - but since I don't know this RemoteFX either, I may just be totally wrong.
Luckily as @josecv21 said updating to Version 2004 solved the issue (making the cause of the problem even more likely to be the windows update), so I guess in the end it all worked out... Thanks for everyone for the quick replies, cheers! :)

**Comment by caseywood** [comment]
> > > > Hi, I have solved my problem.
> > > > For thoses who have the sames issues you have to make 2 things:
> > > > -Activate Virtualization in bios
> > > > -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> > > > & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> > > > It should be work after a reboot.
> > > 
> > > 
> > > 2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.
> > 
> > 
> > Hi. for me the problem was in avast antivirus. In settings click Troubleshoot on the left side of the screen, uncheck the box next to Enable hardware assisted virtualization, then click OK to confirm and restart your computer. it worked for me. Excuse me for my English, I'm from Russia myself 👍 :-)
> 
> Hi, thanks for the reply! I tried it, but unfortunately, it didn't help in my case... :( I suspect it had something to do with the windows update, since just a few days before I used it fine and I don't think anything else changed with in my system...

I had to do the same with AVG

**Comment by pocketkings719** [comment]
I was able to resolve this error by following the steps in this video: https://www.youtube.com/watch?v=VIBdY-5zr58

**Comment by moulie415** [comment]
> > > > > Hi, I have solved my problem.
> > > > > For thoses who have the sames issues you have to make 2 things:
> > > > > -Activate Virtualization in bios
> > > > > -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> > > > > & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> > > > > It should be work after a reboot.
> > > > 
> > > > 
> > > > 2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.
> > > 
> > > 
> > > Hi. for me the problem was in avast antivirus. In settings click Troubleshoot on the left side of the screen, uncheck the box next to Enable hardware assisted virtualization, then click OK to confirm and restart your computer. it worked for me. Excuse me for my English, I'm from Russia myself 👍 :-)
> > 
> > 
> > Hi, thanks for the reply! I tried it, but unfortunately, it didn't help in my case... :( I suspect it had something to do with the windows update, since just a few days before I used it fine and I don't think anything else changed with in my system...
> 
> I had to do the same with AVG

Yes thank you, I'm on AVG and that sorted my problem as well

**Comment by bonellia** [comment]
Just a heads up that the video that @pocketkings719 pointed resolved my issue.
The steps are as follows (for the lazy):

1. In search box type "gpedit" then Goto  - Computer Configuration - Administrative Templates - System - Device Guard - Turn on Virtualization Based Security. Now Double click that and "Disable"

2. In search box, type Turn Windows features on or off, now uncheck Hyper-V and restart system.

3. Open Registry Editor by typing regedit in search box, now Go to HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\DeviceGuard. Add a new DWORD value named EnableVirtualizationBasedSecurity and set it to 0 to disable it.
Next Go to HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\LSA.  Add a new DWORD value named LsaCfgFlags and set it to 0 to disable it.

4. Open command prompt (not powershell) as a administrator and type the following commands  
bcdedit /create {0cb3b571-2f2e-4343-a879-d86a476d7215} /d "DebugTool" /application osloader
bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} path "\EFI\Microsoft\Boot\SecConfig.efi"
bcdedit /set {bootmgr} bootsequence {0cb3b571-2f2e-4343-a879-d86a476d7215}
bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} loadoptions DISABLE-LSA-ISO,DISABLE-VBS
bcdedit /set hypervisorlaunchtype off

Now, Restart your system.

Now I see the following:
```
PS C:\Users\Kril\Downloads\gvm-windows_v1_6_0> .\silent_install_safe.bat

SERVICE_NAME: gvm
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 1  STOPPED
        WIN32_EXIT_CODE    : 0  (0x0)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0
[SC] DeleteService SUCCESS

SERVICE_NAME: gvm
        TYPE               : 1  KERNEL_DRIVER
        STATE              : 4  RUNNING
                                (STOPPABLE, NOT_PAUSABLE, IGNORES_SHUTDOWN)
        WIN32_EXIT_CODE    : 0  (0x0)
        SERVICE_EXIT_CODE  : 0  (0x0)
        CHECKPOINT         : 0x0
        WAIT_HINT          : 0x0
        PID                : 0
        FLAGS              :
PS C:\Users\Kril\Downloads\gvm-windows_v1_6_0>
```

**Comment by bonellia** [comment]
I should add that WSL now doesn't work. I'm now looking to revert these bcdedit commands' effect.

**Comment by mohit9814** [comment]
@bonellia - Dude, I read your comment too late :( 
Please share the commands to revert the bcedit commands.
I need WSL to work

**Comment by Flying--Dutchman** [comment]
@bonellia @mohit9814 
Don't use WSL atm, but I revert the changes simply by removing the bcd and then restoring it with bootrec.
I removed it using EasyBCD.


But maybe you can skip the removal part... You just need the Advanced Startup Options
1. Hit Start
2. Search "BIOS"
3. It will find something like "Advanced Startup" (my system is in german)
4. Click "Restart now"
5. Troubleshoot -> Advanced Options -> Command Prompt
6. type: `bootrec /rebuildbcd`

**Comment by Julio-andre** [comment]
I can't undo the bcd file because I'm using an hide tv screen and I can't see the bios. Anyway, above that my processor hasn't the virtualization option but it has the hypervisor option wich I disable it and I want to use my device, not the simulators of android. what can I do?
I just recently back from "hibernation" and now I'm using windows 10, not a mac or linux sytem. All this is kind of fussing and confusing.

**Comment by Flying--Dutchman** [comment]
@Julio-andre You don't need to open the BIOS, it was just a search query.

Instead of Steps 1-3 you can go open it via the windows settings menu:
"Start" --> "Settings" --> "Update & Security" --> "Recovery"

Again; **Use at your own risk**

**Comment by ljosberinn** [comment]
Even with [the proposed solutions](https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/issues/10#issuecomment-696832752) in this thread, I'm still stuck with the same issue. This is on Win 10 Pro 20H2 / 19042.746. Noteworthy too that this is an almost fresh install (2 weeks old). No difference between using gvm 1.6 or 1.7 too, same error (4294968201).

**Comment by ericoporto** [comment]
This still fails silently for me on Windows 10 when I have Docker running when running Android Studio 4.2 Beta 4 for the first time.

It gets stuck here:

`Running Android Emulator Hypervisor Driver for AMD Processors installer`

**Comment by Merurezki** [comment]
> 
> 
> > > Hi, I have solved my problem.
> > > For thoses who have the sames issues you have to make 2 things:
> > > -Activate Virtualization in bios
> > > -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> > > & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> > > It should be work after a reboot.
> > 
> > 
> > 2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.
> 
> Hi. for me the problem was in avast antivirus. In settings click Troubleshoot on the left side of the screen, uncheck the box next to Enable hardware assisted virtualization, then click OK to confirm and restart your computer. it worked for me. Excuse me for my English, I'm from Russia myself 👍 :-)

hey, it works on me👊🏻👊🏻
thanks!

**Comment by bizh89** [comment]
When I tryied to launch a virtual device (android) on my PC I used guide on this [page](https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/wiki/Is-Hyper-V-really-disabled%3F). But VM does not launch. So I began to use a random method and **turned on Hyper-V** in contrast to guides instructions. And it started.
![image](https://user-images.githubusercontent.com/78523804/106883085-14bdfe00-6712-11eb-84ac-3a8b0eea6d15.png)

**Comment by DPSB** [comment]
I had the same problem, but i was able to solve it after around 2 hours of try and error. 
I couldn't completly disable vbs even when device guard and everything else in the registry or in the group policy showed me that it was disabled. 

So first i checked virtualization status in the BIOS to make sure all required settings are enabled 
( turned out, that SVM Mode & SR-IOV was enabled, but not IOMMU and PCI ARI, so i activated them just to make sure all virtualization components are available and running)
I also checked that Hyper-V, Guarded Host, Virtual Machine Platform & Windows Sandbox was disabled via the optional features. All looked fine but it somehow was still enabled as reported by the silent_install.bat, cpuid & ryzen master (can't start either with vbs enabled) 
Then i found [this](https://superuser.com/questions/1511884/permanently-disable-virtualization-based-security-vbs-in-windows-10-pro) post on superuser, which was pointing to a [microsoft script](https://www.microsoft.com/en-us/download/details.aspx?id=53337) which can be used for checking device guard  readiness, so i thought i might also use it to see if all the (software) components are disabled and it also showed that everything is disabled.
I then noted some registry entry which was showing hypervisorlaunchtype == 1 and because i did not know if 1 was meaning enabled or disabled, i cloned my bootconfig via bcd (because i dont want to change some "stable" setup) and then set the hypervisorlaunchtype to off via bcd, as shown [here](https://stackoverflow.com/a/43843318) well long story short, even that had no effect - it still showed as enabled as reported by cpuid and ryzen master

Since i don't use avast (anymore) i really wasn't to much thinking about windows defender, but infact **it turned out that i had the optional feature "Microsoft Defender Application Guard" enabled**, and when i finally disabled it as mentioned [here,](https://superuser.com/a/1577283) cpuid and also ryzen master finally showed the expected results. 
VBS is now completly disabled and persistent across reboots.

**Comment by ijazahmadkhan888** [comment]
> > > Hi, I have solved my problem.
> > > For thoses who have the sames issues you have to make 2 things:
> > > -Activate Virtualization in bios
> > > -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function"
> > > & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V"
> > > It should be work after a reboot.
> > 
> > 
> > 2days ago my windows 10 updated and even though I had all of the above set and used emulators without any issue, now nothing works and I get this exact error every time. I tried using the newer v1.6 Hypervisor Driver, but got the same results... Any idea of what I can do at this point? I can use my phone and everything else without any issue, but the emulators don't work now.
> 
> Hi. for me the problem was in avast antivirus. In settings click Troubleshoot on the left side of the screen, uncheck the box next to Enable hardware assisted virtualization, then click OK to confirm and restart your computer. it worked for me. Excuse me for my English, I'm from Russia myself 👍 :-)

Love you... Finally it work....

**Comment by abdoutech19** [comment]
**Anyone still having the same issue.**

If you tried everything and still didn't work, type `bcdedit` in the command prompt, and press **ENTER**. If you see an entry for **hypervisorlaunchtype** set to **Auto**, Hyper-V is still enabled. You can try setting the launch type to Off by running the command below.

`bcdedit /set hypervisorlaunchtype off`

That's what worked for me.

**Comment by davidgarciaMontreal** [comment]
@tiko2302, I had the same issue today. Please make sure you have the following features disabled:

https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/wiki/Is-Hyper-V-really-disabled%3F

**Comment by tiko2302** [comment]
> @tiko2302, I had the same issue today. Please make sure you have the following features disabled:
> 
> 
> 
> https://github.com/google/android-emulator-hypervisor-driver-for-amd-processors/wiki/Is-Hyper-V-really-disabled%3F
> 
> 

Yes I did that, thank you. :D
I interpreted an information in my BIOS wrong. It said "SVM: yes" and I thought it meant that the option was enabled. Turns out it just said that the option is supported and was disabled. After I activated it everything worked fine.

**Comment by davidgarciaMontreal** [comment]
@tiko2302 that's great news :) happy coding cheers.

**Comment by dzalf** [comment]
Guys, I just want to confirm that some 'system inspector' applications like Speccy, CPU-Z, etc. might be reporting erroneous values from the CPU and BIOS parameters. Speccy reported having virtualization enabled in the BIOS, however, the software is actually reading inverted values. After setting the paramter SVM (Aus X570-F) to enabled, Speccy reports it as disabled. Upon fixing this, I was able to install the Android Emulator Hypervisor Driver for AMD from Android Studio.

TL;DR: Always make sure that virtualization is ENABLED in your BIOS

**Comment by Ormadont** [comment]
> Hi, I have solved my problem. For thoses who have the sames issues you have to make 2 things: -Activate Virtualization in bios -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function" & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V" It should be work after a reboot.

it's not work for me

**Comment by SpectreWulf** [comment]
So everyone who are still facing this issue in 2022 this is what worked for me finally!

OS Windows 10
CPU: Ryzen 5 5600X

NOTE: WSL won't work if you follow these steps.

Steps:

1. In search box type "gpedit" then Goto -> Computer Configuration -> Administrative Templates -> System -> Device Guard -> Turn on Virtualization Based Security. Now Double click that and "Disable". Apply and Save.
2. In search box, type Turn Windows features on or off, now uncheck Hyper-V and restart system.
3. Open Registry Editor by typing regedit in search box, now Go to HKEYLOCALMACHINE\System\CurrentControlSet\Control\DeviceGuard. If it doesn't exist, add a new DWORD value named EnableVirtualizationBasedSecurity and set it to 0 to disable it (I already had it). 
4. Next Go to HKEYLOCALMACHINE\System\CurrentControlSet\Control\LSA. If it doesn't exis, add a new DWORD value named LsaCfgFlags and set it to 0 to disable it. (Also had it)
5. Open command prompt (not powershell) as a administrator and type the following commands:
- bcdedit /create {0cb3b571-2f2e-4343-a879-d86a476d7215} /d "DebugTool" /application osloader
- bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} path "\EFI\Microsoft\Boot\SecConfig.efi"
- bcdedit /set {bootmgr} bootsequence {0cb3b571-2f2e-4343-a879-d86a476d7215}
- bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} loadoptions DISABLE-LSA-ISO,DISABLE-VBS
- bcdedit /set hypervisorlaunchtype off

Restart system and now it should work when you run "silent_install.bat"

**Comment by zeroarst** [comment]
Is there a solution now for using WSL with this driver...

**Comment by PeRmWaZe** [comment]
I've done everything mentioned here and still nothing. SVM says enabled in my MSI Bios although greyed out so I can't even change it. I went into my Windows Features and everything was already off including Hyper V,  and Virtual machine, and Hypervision. I did all of the Device Guard editing bcedit stuff. My registry already contained those keys and were set 0. I do not have AVG on my PC or any other Virus software and I'm still getting this error...

**Comment by greycr0w** [comment]
> Is there a solution now for using WSL with this driver...

I have been trying to find an answer to this question myself. I need to able to use Docker with an Android Emulator.

**Comment by maxchu92** [comment]
> So everyone who are still facing this issue in 2022 this is what worked for me finally! Make sure you have enabled Virtualization support for your AMD CPU (SVM is enabled in BIOS), then follow the steps below.
> 
> OS Windows 10 CPU: Ryzen 5 5600X
> 
> NOTE: WSL won't work if you follow these steps.
> 
> Steps:
> 
> 1. In search box type "gpedit" then Goto -> Computer Configuration -> Administrative Templates -> System -> Device Guard -> Turn on Virtualization Based Security. Now Double click that and "Disable". Apply and Save.
> 2. In search box, type Turn Windows features on or off, now uncheck Hyper-V and restart system.
> 3. Open Registry Editor by typing regedit in search box, now Go to HKEYLOCALMACHINE\System\CurrentControlSet\Control\DeviceGuard. If it doesn't exist, add a new DWORD value named EnableVirtualizationBasedSecurity and set it to 0 to disable it (I already had it).
> 4. Next Go to HKEYLOCALMACHINE\System\CurrentControlSet\Control\LSA. If it doesn't exis, add a new DWORD value named LsaCfgFlags and set it to 0 to disable it. (Also had it)
> 5. Open command prompt (not powershell) as a administrator and type the following commands:
> 
> * bcdedit /create {0cb3b571-2f2e-4343-a879-d86a476d7215} /d "DebugTool" /application osloader
> * bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} path "\EFI\Microsoft\Boot\SecConfig.efi"
> * bcdedit /set {bootmgr} bootsequence {0cb3b571-2f2e-4343-a879-d86a476d7215}
> * bcdedit /set {0cb3b571-2f2e-4343-a879-d86a476d7215} loadoptions DISABLE-LSA-ISO,DISABLE-VBS
> * bcdedit /set hypervisorlaunchtype off
> 
> Restart system and now it should work when you run "silent_install.bat"

This works for me on my Acer Laptop AMD Ryzen 6800H. Thanks a lot!

**Comment by SpectreWulf** [comment]
> > Is there a solution now for using WSL with this driver...
> 
> I have been trying to find an answer to this question myself. I need to able to use Docker with an Android Emulator.

Unfortunately I am not aware of a method which circumvents around disabling the WSL for the hypervisor to work 😶
Hopefully someone finds this soon!

**Comment by locohamster** [comment]
I had to **ENABLE  SVM MODE** in bios to install this for AMD CPU.

**Comment by GeraldR63** [comment]
Easy going. In my case it was required just to Activate Virtualization in the BIOS of my PC. That' all.

Windows PowerShell
Copyright (C) Microsoft Corporation. Alle Rechte vorbehalten.



PS C:\Users\xyz> Get-WmiObject -Class Win32_BIOS


SMBIOSBIOSVersion : F2
Manufacturer      : Award Software International, Inc.
Name              : Award Modular BIOS v6.00PG
SerialNumber      : 1675433
Version           : GBT    - 42302e31

**Comment by gsevla** [comment]
I guess this issue should be reopened.

As @greycr0w, I need Docker to work and I need Android Emulator too.

Following steps available [here](https://ourcodeworld.com/articles/read/1557/how-to-solve-android-emulator-hypervisor-error-driver-for-amd-processors-installation-failed#disqus_thread), I'm able to get gvm running, but docker stopped working.

Given I need both working in order to get things done, what should I do?

**Comment by antikmozib** [comment]
It worked for me after turning "Memory integrity" off in Windows Security.

![image](https://github.com/google/android-emulator-hypervisor-driver/assets/45801225/83606304-b7be-47fe-ac9c-d1d3b4bb865a)

**Comment by medkhalilbk** [comment]
> Hi, I have solved my problem. For thoses who have the sames issues you have to make 2 things: -Activate Virtualization in bios -Deactivate Hyper-v and hypervisor platform in "activate/desactivate windows function" & run powershell as admin and make :"Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V" It should be work after a reboot.

I tried this and it works, thanks alot!

**Comment by jacobjake1** [comment]
I have hyper-v disabled and Virtulzation on in BIOS. Same thing. Doesn't work.  
![image](https://github.com/user-attachments/assets/861bdf7a-a2d1-49be-b604-1d396ae6c021)

**Comment by Luiyit** [comment]
> It worked for me after turning "Memory integrity" off in Windows Security.
> 
> ![image](https://github.com/google/android-emulator-hypervisor-driver/assets/45801225/83606304-b7be-47fe-ac9c-d1d3b4bb865a)

It worked for me. 
RYZEN 9 3900X - Windows 11
I have Hyper-V disabled, BIOS virtualization on, I'm using Avast

This guide help me a lot to test the service
https://developer.android.com/studio/run/emulator-acceleration?hl=es-419#vm-windows

Thanks @antikmozib

**Comment by LokoAddams** [comment]
> I have hyper-v disabled and Virtulzation on in BIOS. Same thing. Doesn't work. ![image](https://github.com/user-attachments/assets/861bdf7a-a2d1-49be-b604-1d396ae6c021)

I have the same problem, did you achieve to resolve the problem?

**Comment by MertAvci56** [comment]
> > Hyper-v'yi devre dışı bıraktım ve BIOS'ta Virtulzation'ı açtım. Aynı şey. Çalışmıyor.![image](https://github.com/user-attachments/assets/861bdf7a-a2d1-49be-b604-1d396ae6c021)
> 
> Aynı sorun bende de var, sorunu çözebildiniz mi?



> > I have hyper-v disabled and Virtulzation on in BIOS. Same thing. Doesn't work. ![image](https://github.com/user-attachments/assets/861bdf7a-a2d1-49be-b604-1d396ae6c021)
> 
> I have the same problem, did you achieve to resolve the problem?

Is this still going on? How did you solve it? I'm going crazy.

**Comment by LokoAddams** [comment]
> > > Hyper-v'yi devre dışı bıraktım ve BIOS'ta Virtulzation'ı açtım. Aynı şey. Çalışmıyor.![image](https://github.com/user-attachments/assets/861bdf7a-a2d1-49be-b604-1d396ae6c021)
> > 
> > 
> > Aynı sorun bende de var, sorunu çözebildiniz mi?
> 
> > > I have hyper-v disabled and Virtulzation on in BIOS. Same thing. Doesn't work. ![image](https://github.com/user-attachments/assets/861bdf7a-a2d1-49be-b604-1d396ae6c021)
> > 
> > 
> > I have the same problem, did you achieve to resolve the problem?
> 
> Is this still going on? How did you solve it? I'm going crazy.

Yes, in my case I have a DELL laptop and I don't remember what happened but I think DELL laptops have a core isolation that can't be disabled from windows you have to do in the startup, I think I execute a script and reboot. Sorry I tried for 30 minutes to find the link.

**Comment by JeffCodes85** [comment]
I'm on Lenovo Legion Y520 laptop running Windows 10 on an Intel i7-7700HQ.

Mine was fixed after the below:
   In "Turn Windows features on or off":
      Enabled Virtual Machine Platform
      Enabled Windows Hypervisor Platform
   In BIOS, enabled Intel Virtualization
